spec/system/transfer_system_spec.rb

```diff
@@ -135,5 +135,21 @@ def create_transfer(amount, from_name, to_name)
     end
 
     it_behaves_like "Date Range Picker", Transfer
+
+    it "only displays delete button for deletable transfers" do
```

Comment:
Since this doesn't have any interaction, it can be a request spec (which uses a lot less resources).